doc/modules/preprocessing.rst

```diff
@@ -750,11 +751,43 @@ categories are `min_frequency` and `max_categories`.
    input feature. `max_categories` includes the feature that combines
    infrequent categories.
 
-In the following example, the categories, `'dog', 'snake'` are considered
-infrequent::
+In the following example with :class:`OrdinalEncoder`, the categories `'dog' and
+'snake'` are considered infrequent::
 
    >>> X = np.array([['dog'] * 5 + ['cat'] * 20 + ['rabbit'] * 10 +
    ...               ['snake'] * 3], dtype=object).T
+   >>> enc = preprocessing.OrdinalEncoder(min_frequency=6).fit(X)
+   >>> enc.infrequent_categories_
+   [array(['dog', 'snake'], dtype=object)]
+   >>> enc.transform(np.array([['dog'], ['cat'], ['rabbit'], ['snake']]))
+   array([[2.],
+          [0.],
+          [1.],
+          [2.]])
+
+:class:`OrdinalEncoder` can be configured with an encoding unknown categories and
```

Comment:
```suggestion
:class:`OrdinalEncoder` can be configured to encode unknown categories and
```
Not quite sure if this is what you wanted to say, but the original seems to be missing words or have too many?

Reply:
Yea, I agree the original wording was unclear. I reworked the whole paragraph.